src/main/resources/application.properties

```diff
@@ -10,15 +12,11 @@ logging.file=brn.log
 logging.file.max-size=10MB
 logging.file.max-history=15
 
-spring.datasource.url=jdbc:h2:file:C:/brain/brn/h2/brn
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=brn
-spring.datasource.password=brn
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.h2.console.enabled=true
 
 # Vaadin: workaround for https://github.com/vaadin/flow/issues/5947
 spring.devtools.restart.additional-exclude=META-INF/VAADIN/config/**
 
 # Ensure application is run in Vaadin 14/npm mode
 vaadin.compatibilityMode = false
+
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
```

Comment:
could you please add a short comment about what is this property

Reply:
@ElenaSpb 
added link to https://stackoverflow.com/questions/49669628/spring-boot-application-error-with-connect-postgresql-database?noredirect=1&lq=1